components/clarinet-files/src/lib.rs

```diff
@@ -34,14 +34,11 @@ pub type FileAccessorResult<T> = Pin<Box<dyn Future<Output = Result<T, String>>>
 
 pub trait FileAccessor {
     fn file_exists(&self, location: FileLocation) -> FileAccessorResult<bool>;
-    fn read_manifest_content(
+    fn read_file(&self, manifest_location: FileLocation) -> FileAccessorResult<String>;
+    fn read_contracts_content(
         &self,
-        manifest_location: FileLocation,
-    ) -> FileAccessorResult<(FileLocation, String)>;
-    fn read_contract_content(
```

Comment:
Reading contracts or manifest is the same operation from a FS point of view. Let's simplify that